tests/backstop_data/engine_scripts/puppet/mosaico/mailing/editor-test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+module.exports = async (engine, scenario, viewport) => {
+  await require('./step-2.js') (engine, scenario, viewport);
+  await engine.waitFor('a[title="Edit"]');
+  // wait for Angular to bind function
+  await engine.waitFor(200);
+  await engine.click('a[title="Edit"]');
+  await engine.waitFor('.status-start', { hidden: true });
+  await engine.waitFor(() => document.querySelectorAll('body> iframe.ui-front').length > 0);
+
+  const frames = await engine.frames();
+  const frame = frames.find(f => f._navigationURL .match(/civicrm\/mosaico\/iframe\?snippet\=1/g) !== null);
```

Comment:
doesn't this require `await`?

Reply:
No. This doesn't require await. Referenced the code from here - https://github.com/GoogleChrome/puppeteer/issues/618#issuecomment-326066462